Build.PL

```diff
@@ -14,6 +14,13 @@ use builder::custom;
 use File::Basename;
 use File::Spec;
 
+my $tap_harness_args = {
```

Comment:
Why is it necessary to do this in the Build.PL? Then everyone installing this from CPAN would have to install TAP::Formatter::GitHubActions. But we only need this output in the CI, right?

Reply:
> Why is it necessary to do this in the Build.PL? Then everyone installing this from CPAN would have to install TAP::Formatter::GitHubActions. But we only need this output in the CI, right?

It's one of the reasons why I haven't generated a release yet, I want to make the tests pass first, and addressing the generation of build.pl by minilla: (so only on CI we'll need the actions) 

> But we only need this output in the CI, right?

Right :)